src/_App.scss

```diff
@@ -229,6 +247,18 @@ button.dropdown-toggle.nav-button:hover {
   @extend %bottomBorderNav;
 }
 
+// Adjustment for desktop version after adding mobile topnav bar
+.nav-link-container button.dropdown-toggle,
+.nav-link-container button.dropdown-toggle:hover {
+  background: none;
+  color: #404b63;
```

Comment:
should this color be $primary-action-color?

Reply:
This is for text color, which is dark grey. I see it hardcoded in other places in _App.scss so I just follow it.